### ISSUE TITLE
tensorflow chatbot model intent classifier 

### ISSUE THREAD

**Issue by balaram-bhukya** [ugik/notebooks#2]
if you give any random text as input for classify class it will classify as a pre defined class rather it throws null or none value

**Comment by pranjalpandey22** [comment]
Yes this a huge problem! Why is it doing so I have no idea. Wouldn't it be better if there was some way to use a default or a none value?
